### DISCUSSION TITLE
Onyxia's Lair (60) Tuning Feedback

### DISCUSSION THREAD

**Discussion by Annamaria-CC** [chromiecraft/chromiecraft#2898] [PTR Feedback]
This thread is for feedback on Onyxia's Lair (60) regarding it's tuning.
For bugs, please report them [here](https://github.com/chromiecraft/chromiecraft/issues/new/choose)

**Comment by Plasmashield** [comment]
Looks like Onyxian Warder uses 80 version of Flame Lash (4-5k instead of something like 100dmg)
Same for that lava dmg at phase 3
And that Ony's melee dmg just pointless

**Reply by Plasmashield** [reply]
Onyxian Warder now uses correct abilities.
Ony's melee dmg now feels about the same as MC bosses
I think whelps have too much HP. Around 25k against 2.7k in classic (they increased it to 5.4k in SoM). This will cause unnecessary complications in the second phase since this fight is not focused on fighting against whelps. (mostly cuz of aoe diminishing returns)
Also cleave timer seems a bit wrong

**Reply by Annamaria-CC** [reply]
Thank you for your feedback.

We have updated the HP of the whelps. They are still tuned compared to vanilla/tbc. (9k ish). 

About the cleave timer; if you have any proof, please provide it in a bug report, so we can fix it there :)

**Reply by Plasmashield** [reply]
Raid comp is much smaller than it was in classic. Also this is melee fight so it doesn't involve participation in the fight with whelps all dps. So this amount of hp can break mechanics of this boss. But of course, it's much better than what they had before.

**Comment by Hyperbearr** [comment]
Early thoughts from me:

- Tank damage is perhaps a little on the low side in Phase 1 - 600 dtps on a full t1 warrior - could easily be doubled and be fine
- Whelps could do with spawning more frequently but in lower numbers - the WOTLK changes to aoe DR's make the quantity we have untenable compared to vanilla's uncapped AOE
- Eruption doing 3-4k damage per cast to every raid member is insane, some classes barely scrape 4k hp with raid buffs, and most of the low hp classes (in pre-bis damage gear) are melee classes who already lose out on this fight being the big suck for them. Tuning it down to 2-2.5k per raid member feels much more balanced imho. Maybe make her cast it every 15 seconds instead of every 20 to balance?

**Reply by Nyeriah** [reply]
Just to be clear on this one, Eruption is the ground explosion after she fears everyone during the third phase, correct? It already does way less damage than 3-4k.

**Reply by Hyperbearr** [reply]
So I repeated my tests, Seems the issue is that where some of the cracks intersect, you get hit by multiple ticks of it from the multiple cracks, and that just happened to be where I was tanking during testing. Will keep playing with it!

**Comment by Hyperbearr** [comment]
Found another issue (not technically a bug) - the instance being flagged as 10man causes many interface addons to cap raid frames at 10. Might be worth changing the flag to 25?

**Reply by Nyeriah** [reply]
Unfortunatelly there's not much I can do about this. I initially intended to use the 25 map but I ran across a client limitation: level 60 players can't change raid difficulty, the frame that allows changing it only becomes available much later.

**Comment by Plasmashield** [comment]
Bruhhh. It will be nice if you make half as much HP

**Comment by Plasmashield** [comment]
Meleezone looks much smaller than on the retail
https://youtu.be/CokopZzGcKI
So i don't know is that a bug or 80ver difference
Also it look like Onyxia drops too little gold

**Comment by CptGermania** [comment]
So we did a few kills with different tuning and i have to say overall great job, as always.

Here are my notes:

- the threat issue you had after P2 in classic seemed negligible here. Due to how much threat warriors can output now our tank caught up within the first 5 seconds every time
- I agree that the eruption is completely overtuned. If you are full life before the fear, you should not die from one or two ticks. Maybe about 1.5k-2k per tick would make sense imo.
- There are too many whelps coming, but they are so harmless DMG wise. Would prefer fewer whelps with a bit more DMG. generally P2 felt a bit boring as heal (if everyone spreads obviously).
- The fire ball dmg feels perfect, nothing to complain there. Hits hard but doesn't one-shot you if you are full-life.
- P2 also felt a bit draggged out as you lose the dmg from meeles and the healthpool with tuning is considerable
- the breath area seems a lot narrower (less is hit by fire) compared to Classic. Makes this mechanic much easier as there is more space and you are safe earlier.

I did Ony for over 15 month every 5 days on Blizzards Classic. We went from 40 man to 20 man splits after 5 IDs there. How it is now, I think it will stay a bit of a challenge even for a full group, which i think is great. Similarly to MC, i think making it a bit challenging is amazing and i feel that this was achieved. With a well organised group it is very doable (barring any major changes) but i think a few guild will need some extra trys to get the mechanics right.

Again, great job, feel free to ask if anything is unclear.
